docs/orbs-authoring.md

```diff
@@ -3,6 +3,8 @@ This document largely covers the tooling and flow of authoring and publishing yo
 
 Orbs can be authored [inline](inline-orbs.md) in your config.yml file or authored separately and then published to to the orb registry for reuse across projects. This document is largely about how to publish orbs for use across projects. For inline orbs see the [Writing Inline Orbs](inline-orbs.md) document.
 
+**All published orbs can be read and used by anyone, not just members of your organization**. You will have to use inline orbs if you need to write orbs privately (and it is always best to put things that need to be secure into context/environment variables rather than config).
```

Comment:
We should probably put this same warning on the design-considerations.md page - worth repeating in both places.